winterfell/controllers/fileClaimController.js

```diff
@@ -40,71 +41,68 @@ module.exports = function (app) {
     });
   });
 
-  app.post('/claims/create', function (req, res) {
+  app.post('/claims/create', auth.authenticatedOr404, function (req, res) {
     console.log('[createClaim] request received for ' + JSON.stringify(req.body));
-    var extUserId = req.body.extUserId;
     var callback = function (err, claim) {
       if (claim) {
-        res.status(http.OK).send({claim: FileClaim});
+        res.status(http.CREATED).send(claim);
```

Comment:
in general, I like to always send an object as a response.
Consider when one day, we want to add more sibling objects to claim (let's call it `properties`)
So then we want to send()
```
{
  claim: claim,
  properties: props
}
```

We are more prone to backwards compatibility issues if we are coming from
```
.send(claim)
```
since the front-end will have to change. It gets even more hectic if multiple places on the front-end have to change for this. But then again, this is just a website (deploys instantly unlike apps and extensions) and we probably will only have one place to call this endpoint. Just my two cents from past experiences.

TLDR: Can we keep it as `.send({claim: claim})` please? 👻